src/test/testing-library/AuthForm.test.tsx

```
@@ -93,15 +92,18 @@ const fulfilledMockResponse: MockedResponse = {
+interface RenderAuthFormOptions extends Partial<AuthFormWrapperProps> {
+  mocks?: MockedResponse[];
+}
+export function renderAuthForm({
+  onSubmit = mockSubmitSuccess(),
+  mocks = [fulfilledMockResponse],
+  loading = false,
+}: RenderAuthFormOptions = {}): RenderResult {
+  return renderWithProviders(<AuthFormWrapper onSubmit={onSubmit} loading={loading} />, {
+    apolloMocks: mocks,
+  });
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)